### PR TITLE
Django 1.8 support with updated tests

### PR DESCRIPTION
**Pull request by willhcr**
Building on the earlier work by @pedrorodriguesgomes for Django 1.8 support I've updated the test suite code to run under Django 1.7 and 1.8. I also expanded the CI configuration to cover 1.7 and 1.8 as well as Python 3.4. Lastly, I made some minor updates to the README to mention the support for these.

### CODE REVIEW

File: runtests.py [hunk 1]
```diff
@@ -4,9 +4,28 @@
 import sys
 from django.conf import settings
 
+try:
+    from django import setup
```
Comment (benjaoming) [inline runtests.py]: This looks a bit odd :)

I'd suggest

```python
# Detect if django.db.migrations is supported
from django import VERSION as DJANGO_VERSION
NATIVE_MIGRATIONS = (DJANGO_VERSION >= (1, 7))
```

File: runtests.py [hunk 2]
```diff
@@ -40,13 +55,21 @@
...
 
-# If you use South for migrations, uncomment this to monkeypatch
-# syncdb to get migrations to run.
-from south.management.commands import patch_for_test_db_setup
-patch_for_test_db_setup()
+# Native migrations are present in Django 1.7+
+# This also requires initializing the app registry with django.setup()
+# If native migrations are not present, initialize South and configure it for running the test suite
+if NATIVE_MIGRATIONS:
+    setup()
```
Comment (benjaoming) [inline runtests.py]: `from django import setup`
(because I removed it above)

**Comment by benjaoming** [comment]
Are we adding migrations or something? Because it looks like some migration issues are being addressed even though the app currrently doesn't have any :)

**Comment by benjaoming** [comment]
Otherwise looks great, thanks for making all this work finally with tests and all!!

**Comment by willhcr** [comment]
I've added in your recommended changes. Also, the changes to migrations were to avoid this fatal error when testing with Django 1.7+:

```
RuntimeError: South does not support Django 1.7 or higher. Please use native Django migrations.
```

**Comment by benjaoming** [comment]
@willhcr I get it, there are tests that ensure that django-money is compliant with other application's migrations, therefore it's needed... sorry, that was me not getting the point :)

**Pull request merged**